docs/getting-started/integrations-cdn/vuejs-v3.md

```diff
@@ -305,11 +307,11 @@ Since accessing the editor toolbar is not possible until after the editor instan
 ```html
 <template>
     <ckeditor
-			v-if="editor"
-			v-model="data"
-			:editor="editor"
-			@ready="onReady"
-		/>
+		v-if="editor"
+		v-model="data"
+		:editor="editor"
+		@ready="onReady"
+	/>
```

Comment:
The old indention is correct

Reply:
It looks strange, tbh. But you're the boss in Vue :D

Reply:
Indention is off again :smile:

Reply:
Earlier, in this guide, it looks like this:

```html
<template>
	<ckeditor
		v-if="editor"
		v-model="data"
		:editor="editor"
		:config="config"
	/>
</template>
```

Isn't it unified now? Am I stupid? :D

Reply:
It displays like this on GitHub

```html
    <ckeditor
		v-if="editor"
		v-model="data"
		:editor="editor"
		@ready="onReady"
	/>
```

Maybe it's caused by mixing spaces and tabs?

Reply:
Ok, I have my editor configured for tabs. But for some reason, there were some spaces. The snippet was probably off after multiple people copied and pasted it. Now, it should be ok.